src/Workspaces/Core/Portable/CodeGeneration/Symbols/CodeGenerationSymbol.cs

```diff
@@ -71,7 +71,7 @@ internal CodeGenerationSymbol WithAdditionalAnnotations(params SyntaxAnnotation[
 
         public string Language => "Code Generation Agnostic Language";
 
-        public ISymbol ContainingSymbol => null;
+        public virtual ISymbol ContainingSymbol => null;
```

Comment:
Can we go this far?

```suggestion
        public virtual ISymbol ContainingSymbol => throw new NotImplementedException();
```

Reply:
no.  things def call this (but are generally 'null' resilient).  They are not `throw` resilient.  :)    

For example, we commonly pass these symbols into the SymbolDisplay sublayer (which is specifically designed to only assume ISymbols), and that often does some .ContainingSymbol checks.  

That's how i even discovered an issue here, because i tried to just update this base method and broke things :D